website/blog/data-ecosystem/maching-learning-dbt-baton-pass.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "dbt + Machine Learning: What makes a great baton pass?"
+description: "Working with machine learning in dbt can be clunky. Explore three tool paths that makes the baton pass between analytics engineering & machine learning smooth."
+slug: maching-learning-dbt-baton-pass
+
+authors: [sung, izzy]
+
+tags: [data ecosystem]
+hide_table_of_contents: false
+
+date: 2022-02-18
+is_featured: false
```

Comment:
Let's change this to true! The "is_featured" flag determines whether this gets pulled to the top of the feed. We're going to be doing all posts as featured until we get volume up.